documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -122,13 +124,13 @@ include::modules/canceling-migration-ui.adoc[leveloffset=+2]
 [id="migrating-virtual-machines-from-cli"]
 == Migrating virtual machines from the command line
 
-You can migrate virtual machines to {virt} from the command line.
+You can migrate virtual machines to {virt} from the command line. xref:../../../modules/vmware-prerequisites.adoc#vmware-privileges[VMware privileges]
```

Comment:
@sbeskin-redhat what is this additional link? looks like it is broken

Reply:
@mguetta1 
Removed, thank you.
Please approve